dagger/conf.py

```diff
@@ -24,6 +24,7 @@
 EXTERNAL_SENSOR_POKE_INTERVAL = airflow_config.get('external_sensor_poke_interval', 600)
 EXTERNAL_SENSOR_TIMEOUT = airflow_config.get('external_sensor_timeout', 28800)
 EXTERNAL_SENSOR_MODE = airflow_config.get('external_sensor_mode', 'reschedule')
+EVAL = airflow_config.get('evaluate', True)
```

Comment:
Could you give a more descriptive name? Like dummy operator shortcircuit on or something like that?